storage/requirements.txt

```diff
@@ -1 +1,2 @@
-gcloud-aio-auth >= 3.3.0, < 4.0.0
+aiofiles>=0.6.0,<1.0.0
```

Comment:
This looks good for `gcloud-aio`, but for `gcloud-rest` we'll want to [strip it out of the requirements](https://github.com/talkiq/gcloud-aio/blob/193617548e8c3de8d7275cbb6849229a946d6646/.circleci/config.yml#L182).